cloudbuild.yaml

```diff
@@ -0,0 +1,83 @@
+steps:
+
+- name: 'docker/compose'
+  id: Launch Webserver
+  args: ['-f', './docker/openmrs-compose.yaml', '-f', './docker/sink-compose.yml',  'up', '--force-recreate', '--remove-orphan', '-d']
+
+- name: 'gcr.io/cloud-builders/docker'
+  id: Wait for Servers Start
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - e2e-tests/wait-for-start.sh --use_docker_network
+
+- name: 'adoptopenjdk/maven-openjdk8'
+  id: Compile Bunsen and Pipeline
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - mvn --no-transfer-progress -f bunsen install; mvn --no-transfer-progress -f pipelines install
+  waitFor: ['-'] 
```

Comment:
So this will run the wait part in parallel with building (which is very nice), correct?

Reply:
Yup! The whole e2e-test takes 15 minutes to run